enterprise_access/apps/subsidy_requests/admin.py

```diff
@@ -0,0 +1,75 @@
+""" Admin configuration for subsidy_requests models. """
+
+import logging
+
+from django.conf import settings
+from django.contrib import admin
+from django.core.exceptions import ObjectDoesNotExist
+from django.utils.translation import ugettext_lazy as _
+
+from enterprise_access.apps.core.models import User
+from enterprise_access.apps.subsidy_requests.models import (
+    SubsidyRequestCustomerConfiguration,
+)
+from enterprise_access.apps.subsidy_requests.utils import (
+    get_data_from_jwt_payload,
+    get_user_from_request_session,
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+class SubsidyRequestCustomerConfigurationAdmin(admin.ModelAdmin):
+    """ Admin configuration for the SubsidyRequestCustomerConfiguration model. """
+    writable_fields = [
+        'subsidy_requests_enabled',
+        'subsidy_type',
+        'pending_request_reminder_frequency',
+        
+    ]
+    exclude = ['changed_by']
+
+    def get_readonly_fields(self, request, obj=None):
+        """
+        Override to only display some fields on creation of object in admin, as well
+        as limit what is editable after creation.
+        """
+        if obj:
+            return [
+                'enterprise_customer_uuid',
+                'last_changed_by',
+            ]
+        else:
+            return []
+
+    def last_changed_by(self, obj):
+        return 'LMS User: {} ({})'.format(
+            obj.changed_by.lms_user_id,
+            obj.changed_by.email,
+        )
+
+    def save_model(self, request, obj, form, change):
+        """
+        Override save_model method to keep our change records up to date.
+        """
+        current_user = get_user_from_request_session(request)
+        jwt_data = get_data_from_jwt_payload(request, ['user_id'])
+        # Make sure we update the user object's lms_user_id if it's not set
+        # or if it has changed to keep our DB up to date, because we have
+        # no way to predict if the user has hit a rest endpoint and had
+        # their info prepopulated already.
+        lms_user_id = jwt_data['user_id']
+        if not current_user.lms_user_id or current_user.lms_user_id != lms_user_id:
```

Comment:
When would `current_user.lms_user_id != lms_user_id` be true? (Other than in cases where current_user.lms_user_id is null)

Reply:
if the user id changed on the lms side. it _shouldn't_, but was trying to code defensively to make sure we keep what is effectively the user's pk up to date

Reply:
Another weird edge case technically would be if someone changed a User's lms_user_id in the enterprise-access DB via the django admin. this would keep this up to date

Reply:
Ah, ok, so this only applies in the case where:
1. A user has an active session.
2. They're hitting the Django Admin site (which only does session auth, as you noted in another comment).
3. There's an existing `core.User` record based on the auth_user_id in the session cookie.
4. The `lms_user_id` between that `core.User` record and what's in the JWT Auth cookie differ.

Also, your second point is true, but I think you made `lms_user_id` readonly in the Django Admin form, right?